### ISSUE TITLE
broken in latest code insider

### ISSUE THREAD

**Issue by rbadapanda** [iocave/customize-ui#11]
Activity bar is no more showing at bottom of window.

Version: 1.41.0-insider
Commit: bf7d03bf88732af0b46e473c663c49e18e839f9c
Date: 2019-11-15T05:37:27.862Z
Electron: 6.1.4
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 18.7.0

**Comment by knopp** [comment]
I can't really reproduce this (with exact same build). Can you check that you have `"workbench.useExperimentalGridLayout": true` in your settings.json?

**Comment by rbadapanda** [comment]
Ah,  - my bad!

I was cleaning up my VsCode settings files and I noticed that the workbench.useExperimentalGridLayout - was showing up in disabled color. So I removed it.

After putting it back, the extension started working again. Thanks for spending time on this issue..

Just FYNA: the new insider does not show the above setting in the UI anymore. Don't know if they are planning to remove it in future.
